src/main/java/uk/gov/hmcts/reform/sendletter/services/PendingLettersService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sendletter.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sendletter.entity.Letter;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static uk.gov.hmcts.reform.sendletter.entity.LetterStatus.Created;
+
+@Service
+public class PendingLettersService {
+
+    private final LetterRepository repo;
+
+    public PendingLettersService(LetterRepository repo) {
+        this.repo = repo;
+    }
+
+    public List<Letter> getPendingLetters() {
+        return repo
```

Comment:
Based on this code I assume this repository method returns a stream. If that's the case, that stream has to be closed in every case (success or failure), so we need to wrap it in try-with-resources. There's a DB connection and transaction related with it.

Reply:
Also, there will have to be a `@Transactional` annotation somewhere, to make it work.